### ISSUE TITLE
Editing multireddit names doesn't work

### ISSUE THREAD

**Issue by Controless** [ccrama/Slide#606]
I have a multireddit named 'helping_multi' and when I try to change the name to 'helping', an error pops up. I've included some pictures to show this in effect.

![screenshot_2015-11-18-19-38-48](https://cloud.githubusercontent.com/assets/13410769/11261739/38b27436-8e2c-11e5-801f-b9fe473b3fe9.png)
![screenshot_2015-11-18-19-38-55](https://cloud.githubusercontent.com/assets/13410769/11261741/3d71a500-8e2c-11e5-9f92-0f21fc2a7c2c.png)
![screenshot_2015-11-18-19-38-59](https://cloud.githubusercontent.com/assets/13410769/11261744/410ce134-8e2c-11e5-9800-e30a5a4f3870.png)

**Comment by l3d00m** [comment]
This is because it throws a '403 Forbidden' Network Exception. Here's the expection:
```
net.dean.jraw.http.NetworkException: Request returned non-successful status code: 403 Forbidden
11-19 16:17:45.810 14787-14843/me.ccrama.redditslide W/System.err:     at net.dean.jraw.http.RestClient.execute(RestClient.java:132)
11-19 16:17:45.810 14787-14843/me.ccrama.redditslide W/System.err:     at net.dean.jraw.RedditClient.execute(RedditClient.java:150)
11-19 16:17:45.811 14787-14843/me.ccrama.redditslide W/System.err:     at net.dean.jraw.RedditClient.execute(RedditClient.java:144)
11-19 16:17:45.811 14787-14843/me.ccrama.redditslide W/System.err:     at net.dean.jraw.managers.MultiRedditManager.createOrUpdate(MultiRedditManager.java:73)
11-19 16:17:45.811 14787-14843/me.ccrama.redditslide W/System.err:     at me.ccrama.redditslide.Activities.CreateMulti$SaveMulti.doInBackground(CreateMulti.java:275)
11-19 16:17:45.811 14787-14843/me.ccrama.redditslide W/System.err:     at me.ccrama.redditslide.Activities.CreateMulti$SaveMulti.doInBackground(CreateMulti.java:254)
11-19 16:17:45.811 14787-14843/me.ccrama.redditslide W/System.err:     at android.os.AsyncTask$2.call(AsyncTask.java:288)
11-19 16:17:45.812 14787-14843/me.ccrama.redditslide W/System.err:     at java.util.concurrent.FutureTask.run(FutureTask.java:237)
11-19 16:17:45.812 14787-14843/me.ccrama.redditslide W/System.err:     at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
11-19 16:17:45.812 14787-14843/me.ccrama.redditslide W/System.err:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
11-19 16:17:45.812 14787-14843/me.ccrama.redditslide W/System.err:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
11-19 16:17:45.812 14787-14843/me.ccrama.redditslide W/System.err:     at java.lang.Thread.run(Thread.java:818)
```

**Comment by l3d00m** [comment]
It still works with the play store version, so it has something to do with a change we made. It doesn't worked with the commit before my pr though.

**Comment by l3d00m** [comment]
dc93ca006e5551ddf492e809e7be90a747a21922 introduced the bug

**Comment by l3d00m** [comment]
https://github.com/thatJavaNerd/JRAW/issues/93 looks similiar

**Comment by ccrama** [comment]
I believe this is fixed already! Just tested on v4.5
